docs/index.md

```diff
@@ -89,22 +89,19 @@ As a result, only the Events matching the specified query will be returned:
 - one for each subscription to the course
 - one for each subscription of the student
 
-!!! info
-
-    Usually those queries wouldn't be "hard coded". Instead, they can be derived from an in-memory projection (aka "decision model") as demonstrated by some of the [Examples](examples/index.md)
```

Comment:
rephrase: instead of "hard coded" something like "manually write queries, but automatically inferred from decision model definition

Reply:
I added the note again, rephrased as discussed